wildfly-getting-started-archetype/README.adoc

```diff
@@ -0,0 +1,65 @@
+WildFly getting started archetype
+-----------------------------
+
+[[introduction]]
+==== Introduction
+
+This archetype creates a new WAR project which contains a simple Restful web
+service and a HTML page to call the web service.
+
+The generated project is configured to dynamically provision a WildFly server
+and run the web application on the provisoned server so no manual installation
+is required.
+ 
+It is prepared for running Arquillian unit tests.
+More details can be found in the file "src/main/resources/archetype-resources/README.txt", which is end-user doc and added to the resulting project.
+
+[[newwildflyversion]]
+==== New WildFly version
+To update this archetype to a new WildFly version:
+In file "pom.xml":
+
+* update the version
+* update to latest "org.jboss:jboss-parent" version found at https://repo.maven.apache.org/maven2/org/jboss/jboss-parent/
+
+In file "src/main/resources/archetype-resources/pom.xml":
+
+* update the version property named "version.jboss.bom"
+* check whether dependencies have changed.
+* check the plugin versions and update if necessary:
+** wildfly-maven-plugin: https://repo.maven.apache.org/maven2/org/wildfly/plugins/wildfly-maven-plugin/
```

Comment:
Should we use these links or the project thinks? Most maven plugins show he latest version on the top right of their site. Or in the case of the the wildfly-maven-plugin in it's on the left.